playbooks/provision.yml

```diff
@@ -5,4 +5,7 @@
   roles:
   - role: ansible.kubernetes-modules
     install_python_requirements: no
-  - provision-mediawiki123-apb
+  - mediawiki
+  vars:
+    action: provision
+    cluster: openshift
```

Comment:
In the other PR you used `kubernetes` as the default, why the difference?

Reply:
it should be kubernetes